<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ChatInterface</class>
 <widget class="QWidget" name="ChatInterface">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>300</width>
    <height>800</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">#ChatInterface {background-color: qlineargradient(x1:0, y1:0, x2:0, y2:1, stop:0 #FEF6FF, stop:1 #DDF0FF);}</string>
  </property>
  <widget class="QTabWidget" name="tabWidget">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>300</width>
     <height>750</height>
    </rect>
   </property>
   <property name="maximumSize">
    <size>
     <width>1001</width>
     <height>821</height>
    </size>
   </property>
   <property name="styleSheet">
    <string notr="true">QListWidget {background-color: qlineargradient(x1:0, y1:0, x2:0, y2:1, stop:0 #FEF6FF, stop:1 #DDF0FF);}</string>
   </property>
   <property name="tabPosition">
    <enum>QTabWidget::North</enum>
   </property>
   <property name="tabShape">
    <enum>QTabWidget::Rounded</enum>
   </property>
   <property name="currentIndex">
    <number>0</number>
   </property>
   <property name="iconSize">
    <size>
     <width>60</width>
     <height>40</height>
    </size>
   </property>
   <property name="movable">
    <bool>false</bool>
   </property>
   <property name="tabBarAutoHide">
    <bool>false</bool>
   </property>
   <widget class="QWidget" name="tab_friend">
    <attribute name="title">
     <string>好友</string>
    </attribute>
    <attribute name="toolTip">
     <string>好友</string>
    </attribute>
    <attribute name="whatsThis">
     <string>2</string>
    </attribute>
    <widget class="QListWidget" name="listWidget_friend">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>300</width>
       <height>750</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 14pt &quot;微软雅黑&quot;;
QListWidget{  
    background:white;  
    color:black;  
    border:none;  
}  
  
QListWidget::item{  
    border:none;  
        height: 54px;  
}  
  
QListWidget::item:hover{  
    background:rgb(252,240,193)  
}  
  
QListWidget::item:selected{  
    background:rgb(252,233,161);  
    color:black;  
}  
</string>
     </property>
     <property name="dragDropMode">
      <enum>QAbstractItemView::DragOnly</enum>
     </property>
     <property name="resizeMode">
      <enum>QListView::Adjust</enum>
     </property>
     <property name="gridSize">
      <size>
       <width>0</width>
       <height>60</height>
      </size>
     </property>
     <property name="wordWrap">
      <bool>true</bool>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="tab_group">
    <attribute name="title">
     <string>群聊</string>
    </attribute>
    <attribute name="toolTip">
     <string>群聊</string>
    </attribute>
    <attribute name="whatsThis">
     <string>2</string>
    </attribute>
    <widget class="QListWidget" name="listWidget_group">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>300</width>
       <height>750</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 14pt &quot;微软雅黑&quot;;
QListWidget{  
    background:white;  
    color:black;  
    border:none;  
}  
  
QListWidget::item{  
    border:none;  
        height: 54px;  
}  
  
QListWidget::item:hover{  
    background:rgb(252,240,193)  
}  
  
QListWidget::item:selected{  
    background:rgb(252,233,161);  
    color:black;  
}  
</string>
     </property>
     <property name="dragDropMode">
      <enum>QAbstractItemView::DragOnly</enum>
     </property>
     <property name="resizeMode">
      <enum>QListView::Adjust</enum>
     </property>
     <property name="gridSize">
      <size>
       <width>0</width>
       <height>60</height>
      </size>
     </property>
     <property name="wordWrap">
      <bool>true</bool>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>750</y>
     <width>301</width>
     <height>51</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QPushButton" name="pushButton_addfrined">
      <property name="minimumSize">
       <size>
        <width>80</width>
        <height>40</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>80</width>
        <height>40</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <property name="text">
       <string>添加好友</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="pushButton_creategroup">
      <property name="minimumSize">
       <size>
        <width>80</width>
        <height>40</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>80</width>
        <height>40</height>
       </size>
      </property>
      <property name="text">
       <string>创建群聊</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="pushButton_joingroup">
      <property name="minimumSize">
       <size>
        <width>80</width>
        <height>40</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>80</width>
        <height>40</height>
       </size>
      </property>
      <property name="text">
       <string>加入群聊</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
